src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+type TodoFormProps = {
+  onSubmit: (title: string) => void;
+  disabled: boolean;
+};
+
+export const TodoForm: React.FC<TodoFormProps> = ({ onSubmit, disabled }) => {
+  const [title, setTitle] = useState('');
+
+  const handleFormSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    onSubmit(title.trim());
```

Comment:
Before submitting the title, consider checking if the title is not empty after trimming. It will prevent adding empty todos. You can add a condition like this: `if (title.trim()) { onSubmit(title.trim()); setTitle(''); }`.